CONTRIBUTOR-DOCS/04_accessbility_guides/01_semantic_html_aria.md

```diff
@@ -0,0 +1,9 @@
+<!-- Generated breadcrumbs - DO NOT EDIT -->
+
+[CONTRIBUTOR-DOCS](../README.md) / [Accessbility Guide](README.md) / Semantic HTML and ARIA
```

Comment:
This was generated via `node update-nav.js` based on the "Accessiblity guides" folder structure I added.